starknet_devnet/blueprints/rpc.py

```diff
@@ -905,9 +1006,11 @@ def parse_body(body: dict) -> Tuple[Callable, Union[List, dict], int]:
         "getClassHashAt": get_class_hash_at,
         "getClassAt": get_class_at,
         "estimateFee": estimate_fee,
+        "addInvokeTransaction": add_invoke_transaction,
+        "addDeclareTransaction": add_declare_transaction,
+        "addDeployTransaction": add_deploy_transaction,
     }
-
-    method_name = body["method"].lstrip("starknet_")
+    method_name = body["method"].replace("starknet_", "")
```

Comment:
Why did this become necessary?

Reply:
lstrip() works different than expected here.
Here are the [docs](https://python-reference.readthedocs.io/en/latest/docs/str/lstrip.html).

Especially this example shows why the change was necessary:
```
>>> "ABBA".lstrip("AB") # both AB and BA are stripped
''
```

So basically when `starknet_addDeployTransaction` was in `body["method"]`, the result was `ddDeployTransaction`.